crimson/fusioncatcher.py

```diff
@@ -152,11 +166,15 @@ def parse(in_data):
         first_line = src.readline().strip()
         # Parse column names
         colnames = first_line.split("\t")
-        if colnames != _COLS:
+        if colnames == _COLS_0_99_5a:
+            COLS = _COLS_0_99_5a
+        elif colnames == _COLS_1_00:
+            COLS = _COLS_1_00
+        else:
             msg = "Unexpected column names: {0}."
             raise click.BadParameter(msg.format(colnames))
-        for line in (x for x in src):
-            parsed = parse_raw_line(line)
+        for line in src:
+            parsed = parse_raw_line(line, COLS)
```

Comment:
It seems like we can just use `colnames` here and not set `COLS` at all. Something like this, perhaps?

```
if not any([colnames == exp_cols for exp_cols in (_COLS_0_99_5a, _COLS_1_00)]):
    msg = "Unexpected column names: {0}."
    raise click.BadParameter(msg.format(colnames))
for line in src:
    parsed = parse_raw_line(line, colnames)
```

It also makes the check shorter, since what we want to ensure is that the incoming `colnames` is equal to at least one of our supported column names.

What do you think?